### ISSUE TITLE
No handler for "/people/master"

### ISSUE THREAD

**Issue by alemoi** [StarcounterApps/People#134]
### Steps to reproduce
Just check the code `develop` branch.

### Expected result
Handler "/people/master" should exist.

### Actual result
```
"System.NullReferenceException: Object reference not set to an instance of an object.
   at People.MappingHandlers.<>c.<Register>b__0_1() in C:\adfenix-codebase\People\src\People\Api\MappingHandlers.cs:line 21
   at Starcounter.Rest.UserHandlerInfo.RunUserDelegate(Request req, IntPtr methodSpaceUriSpaceOnStack, IntPtr parametersInfoOnStack) in C:\TeamCity\BuildAgent\work\sc-8734\Level1\src\Starcounter.Rest\UriHandlersManager.cs:line 231
   at Starcounter.Internal.Web.AppRestServer.RunDelegateAndProcessResponse(IntPtr methodSpaceUriSpaceOnStack, IntPtr parametersInfoOnStack, Request req) in C:\TeamCity\BuildAgent\work\sc-8734\Level1\src\Starcounter.Apps.JsonPatch\AppRestServer.cs:line 96
   at Starcounter.Rest.UriManagedHandlersCodegen.RunUriMatcherAndCallHandler(String methodSpaceUriSpace, String methodSpaceUriSpaceLower, Request req, UInt16 portNumber, Response& resp) in C:\TeamCity\BuildAgent\work\sc-8734\Level1\src\Starcounter.Rest\UriManagedHandlersCodegen.cs:line 1232
   at Starcounter.Self.DoSelfCall(UInt16 portNumber, String method, String relativeUri, Dictionary`2 headersDictionary, String body, Byte[] bodyBytes, HandlerOptions handlerOptions, Request req) in C:\TeamCity\BuildAgent\work\sc-8734\Level1\src\Starcounter.Internal\Rest\Self.cs:line 390
   at Starcounter.Self.GET(UInt16 port, String uri, Dictionary`2 headersDictionary, HandlerOptions ho) in C:\TeamCity\BuildAgent\work\sc-8734\Level1\src\Starcounter.Internal\Rest\Self.cs:line 133
   at Starcounter.Rest.UserHandlerInfo.<>c__DisplayClass18_2.<RunUserDelegate>b__1() in C:\TeamCity\BuildAgent\work\sc-8734\Level1\src\Starcounter.Rest\UriHandlersManager.cs:line 369
   at Starcounter.Internal.StarcounterEnvironment.RunWithinApplicationInternal(String desiredAppName, Action action) in C:\TeamCity\BuildAgent\work\sc-8734\Level1\src\Starcounter.Internal\Mock\StarcounterEnvironment.cs:line 142
   at Starcounter.Rest.UserHandlerInfo.RunUserDelegate(Request req, IntPtr methodSpaceUriSpaceOnStack, IntPtr parametersInfoOnStack) in C:\TeamCity\BuildAgent\work\sc-8734\Level1\src\Starcounter.Rest\UriHandlersManager.cs:line 368
   at Starcounter.Internal.Web.AppRestServer.RunDelegateAndProcessResponse(IntPtr methodSpaceUriSpaceOnStack, IntPtr parametersInfoOnStack, Request req) in C:\TeamCity\BuildAgent\work\sc-8734\Level1\src\Starcounter.Apps.JsonPatch\AppRestServer.cs:line 96
   at Starcounter.Internal.AppsBootstrapper.ProcessExternalRequest(Request req) in C:\TeamCity\BuildAgent\work\sc-8734\Level1\src\Starcounter.Apps.JsonPatch\AppsBootstrapper.cs:line 348
HResult=-2147467261
"
```

### Technical information
Handler "/people/master" seems not existing for the call:
```cs
            Handle.GET("/people/menu", () =>
            {
                var master = Self.GET<MasterPage>("/people/master");

                master.ShowMenu = false;

                Page page = new Page()
                {
                    Html = "/People/viewmodels/AppMenuPage.html"
                };

                return page;
            });
```

##### Starcounter version 
2.3.1.7294

##### Browsers affected

- [ x] Chrome
- [ x] Firefox
- [ x] Edge
- [ x] Safari

**Comment by atifwaqar** [comment]
I get the same error.
and I have temporarily fixed it by addind following code at line 21 : 
```

**Comment by atifwaqar** [comment]
and by `temporarily fixed` I mean the page doesn't break for my users, and they can log in. but the redirect (from sign-in page to the opened page) doesn't work

**Comment by bouguila** [comment]
this is fixed in [b10885d](https://github.com/StarcounterApps/People/commit/b10885d4646e9313a1fcdf1390ae4959ba352286#diff-ef30e1d709fde34c2cb8648fc7f5889dR29)

**Comment by Mackiovello** [comment]
Closing as fixed. Please reopen if it still doesn't work for you
